modelscope/models/nlp/txl_poem/gpt2/mpu/__init__.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.  All rights reserved.
```

Comment:
this directory is almost identical with the one in mglm, can we reuse them?   and only model implementation code should be kept left in model directory,  please remove other code which are not used